Makefile

```diff
@@ -1,78 +1,60 @@
-# Run local v3 docs site with hot reloading hooked up to es-bs-base and es-ds-components
+# Run local v3 docs site with hot reloading hooked up to es-ds-styles and es-ds-components
 
 .PHONY: dev
-dev:
-	cd es-bs-base; npm link
-	cd es-ds-components; npm link
-	cd es-ds-docs; npm link @energysage/es-bs-base @energysage/es-ds-components
+dev: symlink
 	cd es-ds-docs; npm run dev
 
-# Run the old v2 docs site with slow rebuilds of es-bs-base and es-vue-base
-
-.PHONY: legacy-dev
-legacy-dev:
-	overmind s
+.PHONY: symlink
+symlink:
+	cd es-ds-styles; npm link
+	cd es-ds-components; npm link
+	cd es-ds-docs; npm link @energysage/es-ds-styles @energysage/es-ds-components
 
-# Unlink local v3 docs site from local es-bs-base and es-ds-components; restore to NPM versions
+# Unlink local v3 docs site from local es-ds-styles and es-ds-components; restore to NPM versions
 
 .PHONY: unlink
 unlink:
-	cd es-bs-base; npm unlink @energysage/es-bs-base
+	cd es-ds-styles; npm unlink @energysage/es-ds-styles
 	cd es-ds-components; npm unlink @energysage/es-ds-components
-	cd es-ds-docs; npm unlink --no-save @energysage/es-bs-base
+	cd es-ds-docs; npm unlink --no-save @energysage/es-ds-styles
 	cd es-ds-docs; npm unlink --no-save @energysage/es-ds-components
 	cd es-ds-docs npm install
 
 .PHONY: lint
 lint:
-	npx lerna run lint
+	npm --prefix es-ds-styles run lint
+# TODO: set up linting for es-ds-components and es-ds-docs
+# 	npm --prefix es-ds-components run lint
+# 	npm --prefix es-ds-docs run lint
 
 .PHONY: test
 test:
-	npx lerna run test
+	npm --prefix es-ds-styles run test
+# TODO: set up testing for es-ds-components and es-ds-docs
+#	npm --prefix es-ds-components run test
+#	npm --prefix es-ds-docs run test
 
 .PHONY: build
 build:
-	npx lerna run build
+	npm --prefix es-ds-styles run build
+	npm --prefix es-ds-docs run build
+# es-ds-components does not have a build step
 
 .PHONY: publish
 publish:
```

Comment:
🤔 If we're not going to use lerna, I wonder if we should remove this from the makefile. Using something like release-please (like in `es-storyblok-shared` might be a better solution).

What do you think about removing the `publish` command for now?

Reply:
Yeah that makes sense. All done!